lib/postal.js

```diff
@@ -72,6 +72,7 @@ ChannelDefinition.prototype.publish = function() {
 	if ( typeof envelope !== "object" ) {
 		throw new Error( "The first argument to ChannelDefinition.publish should be either an envelope object or a string topic." );
 	}
+	envelope.headers = {resolverNoCache: _config.resolverNoCache};
```

Comment:
Actually, I think you're right because of this line: `envelope = arguments[ 0 ];` an envelope could be passed in.